app/src/main/java/com/github/sdpteam15/polyevents/database/FirestoreDatabaseProvider.kt

```diff
@@ -163,7 +173,10 @@ object FirestoreDatabaseProvider : DatabaseInterface {
         val ended = Observable<Boolean>()
 
         lastSetSuccessListener = OnSuccessListener<Void> { ended.postValue(true, this) }
-        lastFailureListener = OnFailureListener { ended.postValue(false, this) }
+        lastFailureListener = OnFailureListener {
+            it.message?.let { it1 -> Log.d(this::class.qualifiedName, it1) }
```

Comment:
There is some debug code still here :)

Reply:
Yes the debug code is for us (Maybe we can consider saving these logs?). It would be nice to explain the error to the user, but this should be done in the activities and the fragments where we use the db methods.

Reply:
Sure, we can set something like this, but I think it's better to show a toast on the screen if the goal is to show it to the user, as he can not have access to the logs